src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const [fb, sb] = [Big(numberOne), Big(numberTwo)];
+  switch (operation) {
+    case '-':
+      return fb.minus(sb).toString();
+    case '*':
+      return fb.times(sb).toString();
+    case '+':
+      return fb.plus(sb).toString();
+    case '/':
+      if (sb === '0') return 0;
+      return fb.div(sb).toString();
+    default:
+      return '0';
+  }
+};
```

Comment:
`operate` function is implemented correctly, but you forgot to add the `modulo` case.
```javascript
fb.mod(sb).toString();
```
.....

Reply:
I think that % operant in a calculator is for the percentage instead of module operation and this operation is implemented on the calculate module.

Reply:
@Grifo89  sorry,
I missed it :+1: